dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -152,6 +169,24 @@ def self.regenerate_pdfs(scripts)
       end
     end
 
+    def self.add_metadata_to_ai_s3(file_path, metadata)
+      full_metadata = {
+        metadataAttributes: metadata
+      }
+      File.write(file_path, full_metadata.to_json)
+      flat_filename = File.basename(file_path)
```

Comment:
Per Ceara's guidance we're putting all the files in a "live" subdirectory within the S3 bucket so we have an easier possibility of adding a "staging" version at some point in the future. I also suggest putting a file name prefix for each place we generate these files so that we a) have an easier time seeing where each file is coming from and b) avoid name collisions between the generation methods.

Please adopt this naming convention for the `add_pdf_to_ai_s3` method as well.

```suggestion
      flat_filename = "live/pdfgen_#{File.basename(file_path)}"
```